python/paddle/incubate/autograd/utils.py

```diff
@@ -202,7 +206,12 @@ def prepare_python_api_arguments(op):
         else:
             phi_name = op.type
         inputs, attrs = _get_args_values(op, phi_name)
-        res = [get_var_block(op.block, op.input(n)) for n in inputs]
+        res = []
+        for item in inputs:
+            if item in op.input_names:
+                res.append(get_var_block(op.block, op.input(item)))
+            else:
+                res.append(None)
```

Comment:
This maybe not safe here, but keep it and add a todo to reminds

Reply:
note has been added